package.json

```diff
@@ -11,7 +11,53 @@
     "start": "npm-run-all -p -c server sass:watch",
     "server": "ws --directory test/",
     "sass:compile": "node-sass test/style.scss test/style.css",
-    "sass:watch": "npm-run-all sass:compile \"sass:compile -- -w\""
+    "sass:watch": "npm-run-all sass:compile \"sass:compile -- -w\"",
+    "release": "standard-version -t ''"
+  },
+  "standard-version": {
+    "releaseCommitMessageFormat": "release: {{currentTag}}",
+    "types": [
+      {
+        "type": "build",
+        "hidden": true
+      },
+      {
+        "type": "ci",
+        "hidden": true
+      },
+      {
+        "type": "docs",
+        "section": "Features"
```

Comment:
wrong section for `docs`?

Reply:
Good question. I asked myself, if an extra “Documentation” entry in the changelog is really necessary. Because added docs is always a feature, too. But maybe you’re right. It doesn’t hurt to add all the docs-related changes to a separate section in the changelog. 👍